core/src/components/fab-button/fab-button.tsx

```diff
@@ -153,7 +153,7 @@ export class FabButton implements ComponentInterface, AnchorInterface, ButtonInt
         onClick={(ev: Event) => openURL(this.href, ev, this.routerDirection)}
       >
         <span class="close-icon">
-          <ion-icon name="close" lazy={false}></ion-icon>
+          <slot name="close"></slot>
```

Comment:
We can't merge PRs that introduce breaking changes, @topalavlad solution should work great:

```suggestion
          <slot name="close">
            <ion-icon name="close" lazy={false}></ion-icon>
          </slot>
```

Reply:
@manucorporat my bet is that in any case, one day, this breaking change will happen to remove the dependency on the ion-icons. For example in order to be able to ship this component as a standalone one.

That being said, I've modified the PR as you and @topalavlad suggested.

Reply:
`ionicons` should not be a dependency after this change for `fab-button` if people fill the slot, right? 
It's also probably about when the breaking changes happen, not if they ever do. Probably a major version would be suitable for removing ionicons.

Reply:
ionicons is a different package already but an important part of ionic, i am fine to allow a system to use different icons, but ionions and ionic are part of the same project.